nx/lib/nx.ex

```diff
@@ -6612,6 +6612,152 @@ defmodule Nx do
   defp mean_den(shape, [axis | axes]) when axis >= 0,
     do: elem(shape, axis) * mean_den(shape, axes)
 
+  @doc """
+  Returns the weighted mean for the tensor and the weights.
+
+  If the `:axis` option is given, it aggregates over
+  that dimension, effectively removing it. `axes: [0]`
+  implies aggregating over the highest order dimension
+  and so forth. If the axis is negative, then counts
+  the axis from the back. For example, `axis: [-1]` will
+  always aggregate all rows. You can only aggregate over
+  one axis.
+
+  You may optionally set `:keep_axis` to true, which will
```

Comment:
I'm not sure if we should use keep_axis or keep_axes here. Both would make sense in that keep_axis -> "keep the aggregated axis" and keep_axes -> "keep all axes present in the original tensor"

Reply:
I agree. What do you think @josevalim?

Reply:
We have both `keep_axis` and `keep_axes` in Nx, so I think it should be `keep_axis` when the other option is `axis` (and `keep_axes` for `axes`).